src/main.go

```diff
@@ -107,64 +125,41 @@ func sshHandler(w http.ResponseWriter, r *http.Request) {
 			}
 			return
 		}
-		if messageType == websocket.BinaryMessage || messageType == websocket.TextMessage {
-			_, err = sshIn.Write(p)
-			if err != nil {
-				log.Println("Write to SSH stdin error:", err)
-				return
+		if messageType == websocket.TextMessage {
+			var newSize TerminalSize
+			if err := json.Unmarshal(p, &newSize); err == nil {
+				// Resize terminal
+				//log.Printf("Resizing terminal to cols: %d, rows: %d", newSize.Cols, newSize.Rows)
+				session.WindowChange(newSize.Rows, newSize.Cols)
+			} else {
+				// Write to SSH stdin
+				_, err = sshIn.Write(p)
+				if err != nil {
+					log.Println("Write to SSH stdin error:", err)
+					return
+				}
 			}
 		}
 	}
 }
 
-func checkForVariables() error {
-	if os.Getenv("SSH_USER") == "" {
-		return fmt.Errorf("SSH_USER is not set")
-	}
-	if os.Getenv("SSH_PASS") == "" {
-		return fmt.Errorf("SSH_PASS is not set")
-	}
-	if os.Getenv("SSH_HOST") == "" {
-		return fmt.Errorf("SSH_HOST is not set")
-	}
-	if os.Getenv("SSH_PORT") == "" {
-		return fmt.Errorf("SSH_PORT is not set")
-	}
-	return nil
-}
-
-func printUsage() {
-	fmt.Println("Usage (set proper environmental variables): ")
-	fmt.Println("SSH_USER - username for ssh connection")
-	fmt.Println("SSH_PASS - password for ssh connection")
-	fmt.Println("SSH_HOST - host for ssh connection")
-	fmt.Println("SSH_PORT - port for ssh connection")
-	fmt.Println("MOUNT_HTML - mount html files (default true, set to false to disable)")
-
-}
 func main() {
-	errOf := checkForVariables()
-	if errOf != nil {
-
-		printUsage()
-		log.Fatal(errOf)
-	}
-	shouldMountHTML := os.Getenv("MOUNT_HTML") == "true" || os.Getenv("MOUNT_HTML") == ""
```

Comment:
Why to delete it?

Reply:
I removed it because I thought the HTML always needed to be included in the project, directly within the Docker setup. However, you can keep it if you prefer.